src/ts/components/extensions/richtexteditor/fragments/RichTextEditor.tsx

```diff
@@ -0,0 +1,185 @@
+import { RichTextEditor as MantineRichTextEditor, Link } from "@mantine/tiptap";
+import { Props } from "../RichTextEditor";
+import { useDebouncedValue, useDidUpdate } from "@mantine/hooks";
+import React, { useState, useEffect } from "react";
+
+import { useEditor } from "@tiptap/react";
+import Highlight from "@tiptap/extension-highlight";
+import StarterKit from "@tiptap/starter-kit";
+import Underline from "@tiptap/extension-underline";
+import TextAlign from "@tiptap/extension-text-align";
+import Superscript from "@tiptap/extension-superscript";
+import Subscript from "@tiptap/extension-subscript";
+import Table from "@tiptap/extension-table";
+import TableCell from "@tiptap/extension-table-cell";
+import TableHeader from "@tiptap/extension-table-header";
+import TableRow from "@tiptap/extension-table-row";
+import Placeholder from "@tiptap/extension-placeholder";
+import Color from "@tiptap/extension-color";
+import TextStyle from '@tiptap/extension-text-style';
+import Image from '@tiptap/extension-image';
```

Comment:
Since all these extensions need to be imported in the component, is there any reason not to enable them all by default?  Is there a performance difference?

Reply:
All good points.  Let's leave it as-is. :heavy_check_mark: